commands/git/git-push.md

```diff
@@ -9,11 +9,20 @@
 
 `git push origin newbranch`
 
-### Demo
+#### Difference between:
+`git push --force` & `git push --force-with-lease`
+
+`force` overwrites a remote branch with your local branch. <br>
+
+`--force-with-lease` is a safer option that will not overwrite any work on the remote branch if more commits were added to the remote branch (by another team-member working on the same repository). It ensures you do not overwrite someone elses work by force pushing.
 
-<img src="../../gifs/git-push.gif" alt="Git Push"/> <br>
+[Click here to see the orginial git commit on the same.](https://github.com/git/git/commit/28f5d176110d2ed768a0a49159993c7a02d8cb15)
```

Comment:
Let's remove this line. The main purpose of this repository is to simplify the details of the commands, this link provides a lot of information.